### PR TITLE
fix(core): edit tag should not jump to its detail

### PR DESCRIPTION
**Pull request by pengx17**
fix AF-1324

### CODE REVIEW

**Comment by pengx17** [comment]
* **#8070** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8070?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pengx17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/components/page-list/operation-cell.tsx [hunk 1]
```diff
@@ -492,7 +492,10 @@ export const TagOperationCell = ({
       <IconButton
         tooltip={t['Rename']()}
         tooltipOptions={tooltipSideTop}
-        onClick={() => setOpen(true)}
+        onClick={e => {
```
Comment (forehalo) [inline packages/frontend/core/src/components/page-list/operation-cell.tsx]: one more

File: packages/frontend/core/src/components/page-list/operation-cell.tsx [hunk 2]
```diff
@@ -492,7 +492,13 @@ export const TagOperationCell = ({
       <IconButton
         tooltip={t['Rename']()}
         tooltipOptions={tooltipSideTop}
-        onClick={() => setOpen(true)}
+        onClick={useCallback(
+          (e: React.MouseEvent<HTMLButtonElement>) => {
+            e.preventDefault();
```
Comment (forehalo) [inline packages/frontend/core/src/components/page-list/operation-cell.tsx:497]: I mean this

**Comment by JimmFly** [comment]
It looks like the z-index of this modal is wrong.
CleanShot 2024-09-03 at 19 01 09@2x

File: packages/frontend/core/src/components/page-list/operation-cell.tsx (hunk 2 above)
Comment (pengx17) [reply packages/frontend/core/src/components/page-list/operation-cell.tsx:497]: I am sorry?

Comment (forehalo) [reply packages/frontend/core/src/components/page-list/operation-cell.tsx:497]: useCatchEventCallback

Comment (pengx17) [reply packages/frontend/core/src/components/page-list/operation-cell.tsx:497]: this is preventDefault, not stoppropagation

**Comment by pengx17** [comment]
> It looks like the z-index of this modal is wrong.
> CleanShot 2024-09-03 at 19 01 09@2x
> 

I think this is caused by other issues

**Review by forehalo — APPROVED** [review]

**Pull request merged**